modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js

```diff
@@ -60,9 +61,7 @@ function OrdersListView({
 						if (!items || !items.length) {
 							return (
 								<EmptyListView
-									caption={Liferay.Language.get(
-										'no-orders-were-found'
-									)}
+									caption="no-orders-were-found"
```

Comment:
@gianmarcobrunialti does it mean, here we will not have anymore localization

Reply:
@ivicac For the `EmptyListView` component, i18n is centralized.
![image](https://user-images.githubusercontent.com/49437061/111302547-d4786680-8653-11eb-8551-d37eebb62e4b.png)